scripts/deploy.sh

```diff
@@ -0,0 +1,12 @@
+# This script runs the aws cli command to deploy the app to s3
+# It also uploads source maps to bugsnag
+
+# Try this step and throw an error if it fails
+echo "Deploying to S3"
+aws s3 sync build/ s3://${BUCKET}/ --acl public-read --follow-symlinks --delete 
```

Comment:
I think you may have already addressed this in your response to me but I wasn't sure HAHA just wanted to flag that Parsley has this extra thing! It might not be necessary at all but just wanted to point it out in case it is.
```suggestion
aws s3 sync build/ s3://${BUCKET}/ --acl public-read --follow-symlinks --delete --exclude .env-cmdrc.json 

```